src/middlewared/middlewared/plugins/failover_/event.py

```diff
@@ -644,6 +644,7 @@ def vrrp_master(self, job, fobj, ifname, event):
                     'However, other zpools imported so the failover process continued.'
                 )
         else:
+            self.middleware.create_task(self.middleware.call('zfs.pool.ddt_prefetch_pools'))
```

Comment:
This will fail on pools that have the root dataset encrypted. I'd rather move this call to further down in the failover event. I'd like to actually call this after we start the critical services (NFS, SMB and iSCSI)

Reply:
We do at this point have root datasets unlocked already but yes i'll move it to after we have restarted critical services ✅